docs/HACKING.md

```diff
@@ -33,7 +33,9 @@ for testing purposes.
 First, install `mkosi` from the
 [GitHub repository](https://github.com/systemd/mkosi#running-mkosi-from-the-repository).
 Note that it's not possible to use your distribution's packaged version of mkosi
-as mkosi has to be installed outside of `/usr` for the following steps to work.
+as mkosi has to be installed outside of `/usr` for the following steps to work. When
+installing mkosi from the github repository, make sure to symlink it to `~/.local/bin`
+instead of `/usr/local/bin` if you want to add it to your `$PATH`.
```

Comment:
Is it necessary to mention /usr/local/bin ?? Why not to just mention that the .local must be in `$PATH` ?

Reply:
@yuwata In the mkosi docs we used to recommend people symlink it to /usr/local/bin so I wanted to emphasize that doesn't work anymore.

Reply:
@yuwata if it's really true that it doesn't work anymore, I agree it's worth mentioning b/c it would be pretty subtle to somehow figure that out if it changed after a build was already set up. For me everything I've tried so far worked the same before I changed the path.

@DaanDeMeyer I was under the impression that the primary reason for installing mkosi directly from its github repo was that systemd/main builds typically require mkosi features that aren't in most distro's curated versions of mkosi yet. I believe this used to be mentioned in this doc.

Reply:
OK. Then, let's merge this as is. Further improvements can be done later.